build.fsx

```diff
@@ -119,21 +114,16 @@ Target "CopyBinaries" (fun _ ->
 // Clean build results
 
 Target "Clean" (fun _ ->
-    !! solutionFile |> MSBuildRelease "" "Clean" |> ignore
+    // !! solutionFile |> MSBuildRelease "" "Clean" |> ignore
     CleanDirs ["bin"; "temp"; "docs/output"]
 )
 
 // --------------------------------------------------------------------------------------
 // Build library & test project
 
 Target "Build" (fun _ ->
-    !! solutionFile
-#if MONO
-    |> MSBuildReleaseExt "" [ ("DefineConstants","MONO") ] "Build"
-#else
-    |> MSBuildRelease "" "Build"
-#endif
-    |> ignore
+    !! solutionFile |> MSBuildRelease "" "Restore" |> ignore
+    !! solutionFile |> MSBuildRelease "" "Build" |> ignore
```

Comment:
We should probably use the `DotNetCli` module here to run the restore and build, which should be one task with that module.

Reply:
I was reverting back to MSBuild for travis to be happy. See this CI run: https://travis-ci.org/fsprojects/FSharpx.Extras/builds/345267943?utm_source=github_status&utm_medium=notification

I would assume this can only be "fixed" with only compiling to netstandard.
Any opinions on dropping net45?